README.md

```diff
@@ -289,18 +289,40 @@ Just as with React, but assuming Angular instead :)
 
```

Comment:
Add a comment explaining that the example will illustrate how to provide telescopes to components using dependency injection and inputs consumed through templates.

Reply:
Would you like me to keep your example - and add another example - as `Angular DI` ? since there is multiple ways to work with `Telescope`